.github/workflows/ci-flake8.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     strategy:
       max-parallel: 4
       matrix:
-        python-version: [3.9]
+        python-version: [3.12]
```

Comment:
I notice the Dockerfile builds off Python 3.10. Any particular reason for choosing different versions? The CI actions can presumably be set to try both.

Reply:
No, no reason, I just haven't kept them all aligned when I've updated it. I'll put them up to 12